SimTKcommon/Geometry/src/DecorativeGeometryRep.h

```diff
@@ -568,12 +568,20 @@ class DecorativeMeshFileRep : public DecorativeGeometryRep {
 // no default constructor
 explicit DecorativeMeshFileRep(const std::string& meshFileName) 
 :   meshFile(meshFileName) {
+    pmesh = nullptr;
 }
 
 const std::string& getMeshFile() const {
     return  meshFile;
 }
 
+const PolygonalMesh& getMesh() {
```

Comment:
> I wasn't sure which one is cleaner (const and mutable or neither)

I'm happy with either as long as there is a comment explaining the intent. You could leave it as-is and add a comment like `/* This is logically const but non-const here to facilitate caching of the mesh once opened. */`. Or, change it to const and write `/* Although this is const, it saves the mesh in a mutable cache the first time it is called. */`.